README.md

```diff
@@ -272,7 +319,7 @@ The most common ones we use are size (to change the number of results returned)
 
 When using q, you'll need to use the [Lucene Query Parser Syntax](http://lucene.apache.org/core/2_9_4/queryparsersyntax.html).
```

Comment:
Funny that I was struggling with some of this stuff today and didn't realize it was in the docs here. :grimacing: